### DISCUSSION TITLE
How do you know if your school has force enterprise enrollment on your school Chromebook?

### DISCUSSION THREAD

**Discussion by Su4o** [3kh0/ext-remover#796] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
as above. Can i check in chrome://policy? How can i check?

**Comment by 3kh0** [comment]
> as above. Can i check in chrome://policy? How can i check? 

I am pretty sure it is all in there, just look at the names, I am sure you can figure it out. Most likely it is enabled

**Comment by Su4o** [comment]
checked all policies and i still can't find it. Know any other way?

**Comment by Hilololsusy** [comment]
Just back up youre cookies and test it.

**Comment by AshtonDavies** [comment, marked as answer]
It doesn't hurt to test it yourself by actually doing it.

**Comment by rwg100** [comment]
power wash your Chromebook and if it reenrolls itself then it has it just back your stuff up before you do that

**Comment by Su4o** [comment]
Problem is that even if it does work, (unenrollment) I want to re-enroll if it is un-enrolled or use FakeMurk. If it works, it could just erase the Chromebook HWID of ever enrolling to my school. Thus, no re-enroll making me in trouble. That's kinda the problem.

**Comment by Su4o** [comment]
Ok. Sadly realised this, but fakemurk doesn't work with ver 118. on chromeos ver 116 and under. Anyone knows any way to hide the fact that your un-enrolled when enrolling on school chromebook (alternatives to fakemurk?)
